src/orb.yml.hbs

```diff
@@ -848,42 +848,51 @@ commands:
           Only in use when verify-revision-is-deployed is set to true.
         type: boolean
         default: true
+      run-update-task-definition:
```

Comment:
I'm thinking it might be better to rename this to `skip-task-definition-registration` (and retain `false` as the default value as registering a new task definition is likely to be a more common use case). Then the `description` could be changed to something like:

```
Whether to skip registration of a new task definition.
```

Reply:
Would you also be able to update the `deploy-service-update` job so that it uses the new parameter?